bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/admin/TableAdapter.java

```diff
@@ -41,37 +44,34 @@
       ColumnDescriptorAdapter.INSTANCE;
   protected final BigtableInstanceName bigtableInstanceName;
 
-
-
   /**
    * <p>adapt.</p>
    *
-   * @param desc a {@link org.apache.hadoop.hbase.HTableDescriptor} object.
-   * @return a {@link com.google.bigtable.admin.v2.Table} object.
+   * This method adapts ColumnFamily to CreateTableRequest.
+   *
+   * @param  createTableRequest a {@link CreateTableRequest}
+   * @param desc a {@link HTableDescriptor} object.
    */
-  protected static Table adapt(HTableDescriptor desc) {
-    Map<String, ColumnFamily> columnFamilies = new HashMap<>();
+  @InternalApi
+  protected static void adapt(CreateTableRequest createTableRequest, HTableDescriptor desc) {
```

Comment:
Please move the output arg last so that it read adapt something input something else